actors/builtin/multisig/multisig_actor.go

```diff
@@ -368,11 +367,30 @@ func (a Actor) approveTransaction(rt vmr.Runtime, txnID TxnID, proposalHash []by
 }
 
 func (a Actor) validateSigner(rt vmr.Runtime, st *State, address addr.Address) {
-	if !st.isSigner(address) {
+	if !a.isSigner(rt, *st, address) {
 		rt.Abortf(exitcode.ErrForbidden, "party not a signer")
 	}
 }
 
+func (a Actor) isSigner(rt vmr.Runtime, st State, address addr.Address) bool {
```

Comment:
Pass the state by pointer, not by copy. The mutation below will have no effect otherwise.

Reply:
D'oh, sorry!